test/e2e_node/conformance/build/Makefile

```diff
@@ -0,0 +1,94 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build the node-test image.
+#
+# Usage:
+#   [ARCH=amd64] [REGISTRY="gcr.io/google_containers"] [BIN_DIR="../../../../_output/bin"] make (build|push) VERSION={some_version_number e.g. v0.1}
+
+# TODO(random-liu): Add this into release progress.
+REGISTRY?=gcr.io/google_containers
+ARCH?=amd64
+# BIN_DIR is the directory to find binaries, overwrite with ../../../../_output/bin
+# for local development.
+BIN_DIR?=../../../../_output/dockerized/bin/linux/${ARCH}
+TEMP_DIR:=$(shell mktemp -d)
+
+UNAME_S:=$(shell uname -s)
+ifeq ($(UNAME_S),Darwin)
+	SED_CMD?=sed -i ""
+endif
+ifeq ($(UNAME_S),Linux)
+	SED_CMD?=sed -i
+endif
+
+ifeq ($(ARCH),amd64)
+       BASEIMAGE?=debian:jessie
```

Comment:
I've tried busybox, but because the test is dynamically linked and relies on some library. It turns out that using debian becomes simpler.

I've also tried to compile with static link, however, the function [`user.Current()`](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/remote/remote.go#L58) we are using can not be compiled with `CGO_ENABLED=0`. http://stackoverflow.com/questions/29898620/user-current-not-implemented-on-linux-amd64-with-golang-using-docker-scrtach